src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -637,6 +645,10 @@ class TriggerView extends Component<TriggerViewProps, TriggerViewState> {
             })
     }
 
+    onClickManifestDownload = (appId: number, envId: number) => {
+        getDeployManifestDownload(appId, envId, this.setManifestLoader)
```

Comment:
there should be a check for getDeployManifestDownload

Reply:
Done